src/main/java/com/google/sps/data/Project.java

```diff
@@ -0,0 +1,54 @@
+package com.google.sps.data;
+
+/** Class containing projects*/
```

Comment:
Does it contain the project or does it represent the project?

Reply:
It represents a project.